containerd-shim/main.go

```diff
@@ -71,11 +72,7 @@ func start() error {
 	if err != nil {
 		return err
 	}
-	defer func() {
-		if err := p.Close(); err != nil {
-			logrus.Warn(err)
-		}
-	}()
+	defer p.Close()
```

Comment:
Yeah, but I think better for them to be here instead of having to add them later if we hit a weird bug. @tonistiigi wdyt?